CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ option(USE_GPU_BRIDGE "Enable the GPU bridge (GPU/AMGCL solvers)" ON)
 option(USE_TRACY_PROFILER "Enable tracy profiling" OFF)
 option(CONVERT_CUDA_TO_HIP "Convert CUDA code to HIP (to run on AMD cards)" OFF)
 set(OPM_COMPILE_COMPONENTS "2;3;4;5;6;7" CACHE STRING "The components to compile support for")
+option(OPM_ENABLE_OPENCL "Enable OpenCL support?" ON)
```

Comment:
please use USE_OPENCL for consistency with other such options.

Reply:
Fixed. I took inspiration from `OPM_ENABLE_PYTHON` I think.

Reply:
sounds likely. i'd really like to change that as well but it has lived too long now so i'd break too much...